uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1,4 +1,57 @@
 package NeuroDB::MRIProcessingUtility;
+
+
+=pod
+
+=head1 NAME
+
+NeuroDB::MRIProcessingUtility -- Provides an interface for MRI processing
+utilities
+
+=head1 SYNOPSIS
+
+  use NeuroDB::ProcessingUtility;
+
+  my $utility = NeuroDB::MRIProcessingUtility->new(
```

Comment:
indentation is awkward in the SYNOPSIS section; in case you are want them to look perfect when running perldoc :)